CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# 👋 Contribution guidelines
+
+This document describes how to contribute to Revenge.
```

Comment:
Should this refer to Revenge bundle or Revenge? The other repos refer to Revenge Manager or Xposed respectively.

Reply:
I'm not really sure. What do you think?

Reply:
I'd say bundle so that it is consistent with the other reops and true to its name